src/server/service/kv.rs

```diff
@@ -318,6 +318,47 @@ impl<T: RaftStoreRouter + 'static, E: Engine> tikvpb_grpc::Tikv for Service<T, E
         ctx.spawn(future);
     }
 
+    fn kv_refresh_lock(
+        &mut self,
+        ctx: RpcContext<'_>,
+        mut req: RefreshLockRequest,
+        sink: UnarySink<RefreshLockResponse>,
+    ) {
+        let timer = GRPC_MSG_HISTOGRAM_VEC.kv_refresh_lock.start_coarse_timer();
+
+        let (cb, f) = paired_future_callback();
```

Comment:
better to extract a `future_refresh_lock` to be consistent with other functions.

Reply:
To do that, I will modify kvproto first to support batch commands. https://github.com/pingcap/kvproto/pull/414